src/controllers/dynakube/version/codemodules.go

```diff
@@ -57,7 +57,7 @@ func (updater *codeModulesUpdater) CheckForDowngrade(latestVersion string) (bool
 	return false, nil
 }
 
-func (updater *codeModulesUpdater) UseTenantRegistry(_ context.Context, _ *dockerconfig.DockerConfig) error {
+func (updater *codeModulesUpdater) UseTenantRegistry(_ context.Context, _ string, _ *dynatracev1beta1.DynaKube, _ client.Reader) error {
 	customVersion := updater.CustomVersion()
 	if customVersion != "" {
 		updater.dynakube.Status.CodeModules = dynatracev1beta1.CodeModulesStatus{
```

Comment:
why do we pass in a `dynakube`
if the `updater` already has a `dynakube` 😅 
(I think the same could be done for the `apiReader`)

Reply:
True. apiReader must be passed still, as the updater only contains the dtClient AFAIK

Reply:
you could add to the struct of the updater

Reply:
:+1: [I just did it](https://github.com/Dynatrace/dynatrace-operator/pull/1998/commits/9f11156912abb79ddfcf81b6048b51bfc8055fbf)